src/components/ui/Modal/modal.scss

```diff
@@ -1,6 +1,6 @@
 .Layer__ModalOverlay {
   position: fixed;
-  z-index: 100;
+  z-index: 1001;
```

Comment:
I think this is fine for now- When less time-sensitive, can you, on a separate PR, research the following:

1. What conventions does the project use in terms of z-index? (use AI, critique existing structure)
2. Can you research/propose z-index hierarchy for the different components? (use AI, recommend, then critique recommendation)

Then maybe this proposal would be a great way to start the conversation on z-indexes, and can be something we follow on your lead!

Reply:
Key regression to avoid:

1. Modal is now set to `z-index: 1001`
2. On the modal is a `Span` with a tooltip.
3. What `z-index` does the tooltip currently use? If less than 1001, then this will cause a regression. Expected: Tooltip on Modal is higher than Modal.